finality-aleph/src/aggregation/aggregator.rs

```diff
@@ -0,0 +1,146 @@
+use crate::aggregation::{Hash, Multicast, Multisigned, SignableHash};
+use crate::{metrics::Checkpoint, network::DataNetwork, Metrics};
```

Comment:
it is strange that even though we have `imports_granularity = "Crate"` in our `rustfmt.toml` it doesn't merge these